lib/helpers/ajaxHelper.js

```diff
@@ -0,0 +1,29 @@
+const $ = require('jquery');
+const setUrl = require('../models/apiUrl').setUrl;
+
+class AjaxRequests {
```

Comment:
Awesome refactor, @chantal66! Keeping all of the ajax requests in one file seems like a much more clear architecture that should be much easier to work with in the future.